packages/ckeditor5-source-editing/docs/features/source-editing.md

```diff
@@ -55,6 +55,20 @@ ClassicEditor
 	.catch( ... );
 ```
 
+To achieve Markdown source editing mode just add the MArkdown plugin to the editor.
```

Comment:
There is a typo in the plugin name.
Also I think it would be nice to link the plugin name with its API docs.

```suggestion
To achieve Markdown source editing mode just add the {@link module:markdown-gfm/markdown~Markdown} Markdown plugin to the editor.
```